easybuild/easyconfigs/f/FFLAS-FFPACK/FFLAS-FFPACK-2.2.0-foss-2016a.eb

```diff
@@ -0,0 +1,46 @@
+##
+# This file is an EasyBuild recipe; see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright (c) 2016 Riccardo Murri <riccardo.murri@gmail.com>
+# Authors::   Riccardo Murri <riccardo.murri@gmail.com>
+# License::   GPL
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'FFLAS-FFPACK'
+version = '2.2.0'
+
+homepage = 'https://linbox-team.github.io/fflas-ffpack/'
+description = "Finite Field Linear Algebra Subroutines / Package"
+
+toolchain = {'version': '2016a', 'name': 'foss'}
+
+sources = ['v2.2.0.zip']
+source_urls = ['https://github.com/linbox-team/%(namelower)s/archive']
+
+builddependencies = [
+    ('Autoconf', '2.69', '', True),
+    ('Automake', '1.15', '', True),
+    ('libtool', '2.4.6', '', True),
+]
+dependencies = [
+    ('GMP', '6.1.0', '', ('GCC', '4.9.3-2.25')),
+    ('Givaro', '4.0.1'),
+]
+
+preconfigopts = "env NOCONFIGURE=1 ./autogen.sh && "
+configopts = '--with-gmp=$EBROOTGMP --with-givaro=$EBROOTGIVARO --with-blas-cflags="-I$BLAS_INC_DIR" --with-blas-libs="-L$BLAS_LIB_DIR $LIBBLAS" --enable-openmp'
+
+parallel = 4
```

Comment:
any particular reason for this? the ideal option is system-dependent (and EB does a good job at deriving a good value based on the available resources)

If you want to limit it to 4 cores max, use `maxparallel = 4`